.github/workflows/cli-tests.yaml

```diff
@@ -99,19 +99,13 @@ jobs:
             brew install dash zsh
           fi
       - name: Install Nix
-        run: |
-          sh <(curl -L https://nixos.org/nix/install) --no-modify-profile --daemon --daemon-user-count 1
-          . /nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh
-          export NIX_PATH=nixpkgs=https://github.com/nixos/nixpkgs/tarball/nixos-unstable
-          nix-build '<nixpkgs>' -A stdenv -A bash -A hello
-          echo "__ETC_PROFILE_NIX_SOURCED=1" >> $GITHUB_ENV
-          echo "NIX_PROFILES=$NIX_PROFILES" >> $GITHUB_ENV
-          echo "NIX_SSL_CERT_FILE=$NIX_SSL_CERT_FILE" >> $GITHUB_ENV
-          echo "NIX_PATH=$NIX_PATH" >> $GITHUB_ENV
-          echo "PATH=$PATH" >> $GITHUB_ENV
+        uses: DeterminateSystems/nix-installer-action@v1
+        with:
+          logger: pretty
+          nix-build-user-count: 4
```

Comment:
is there a reason behind putting 4 here or it's just to have a multi-user mode

Reply:
It defaults to 32, but creating 32 user accounts on macOS can be slow so I reduced it to 4. I don't think it should affect things too much since none of our jobs should be building from source.